tests/internal/test_database_monitoring.py

```diff
@@ -97,6 +97,10 @@ def test_dbm_propagation_full_mode():
 
     dbspan = tracer.trace("dbname", service="orders-db")
 
+    # since inject() below will call the sampler we just call the sampler here
+    # so sampling priority will align in the traceparent
+    tracer.sampler.sample(dbspan._local_root)
```

Comment:
Gotcha, for this case, in the real world it should always covered by the dbm inject code: https://github.com/DataDog/dd-trace-py/pull/8308/files#diff-b86d5de5f005bdd3a34c46580f7bb4310b4a2925cdb907a3e1c35deffc88c6a2R65-R69. We just have to call it early in this test so that when we do `sqlcomment = dbm_popagator._get_dbm_comment(dbspan)` it grabs a comment with a traceparent that has a priority. 

Another option would be to sample inside of `_get_dbm_comment` but that felt worse to me initially.